operator/v1alpha1/types.go

```diff
@@ -97,6 +99,10 @@ type OperatorStatus struct {
 	// observedGeneration is the last generation change you've dealt with
 	ObservedGeneration int64 `json:"observedGeneration,omitempty" protobuf:"varint,1,opt,name=observedGeneration"`
 
+	// operatorVersion indicate what level of the operator was used to realize this status.  This knowledge feeds back into
+	// decisions about whether or not to force a workload update since the workload generation will not have changed.
+	OperatorVersion string `json:"operatorVersion" protobuf:"bytes,7,opt,name=operatorVersion"`
```

Comment:
optional or required?

Reply:
> optional or required?

Logically required, but flagging it as required does bad things to protobuf when skewing versions